packages/server/src/AccountsServer.js

```diff
@@ -302,6 +321,57 @@ export class AccountsServer {
     const res = await this.db.setProfile(userId, { ...user.profile, ...profile });
     return res;
   }
+
+  async sendVerificationEmail(userId: string, address: string): Promise<void> {
+    const user = await this.db.findUserById(userId);
```

Comment:
It is not exactly the same things for `sendVerificationEmail` this will return the first unverified email.
For `sendResetPasswordEmail` it will return the first email.